ml/pre-processing/PreProcessingConfig.scala

```diff
@@ -1,9 +1,9 @@
-object PreProcessingConfig {
-
-  val keyspacekeyspace = "gsoc3"
+object PreProcessingConfig extends Serializable{
```

Comment:
I would recommend merging this into your config. You have two easy options as you are using a standard config format:
1) find a library for scala that works with your config format. 
2) convert your config to JSON and then use the JSON style configuration for both your scala and python code.